plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/interpretator/MacInterpreterDownloader.kt

```diff
@@ -0,0 +1,66 @@
+package io.paddle.plugin.python.dependencies.interpretator
+
+import io.paddle.project.PaddleProject
+import java.io.File
+
+internal class MacInterpreterDownloader(userDefinedVersion: InterpreterVersion, project: PaddleProject) :
```

Comment:
I think it would be cool to run some tests on GH Actions CI for macos-latest, ubuntu-latest later which will trigger at least some of these behaviors 🤔

Reply:
I have it could be a part TestContainer workflow